src/student.rb

```diff
@@ -1,7 +1,9 @@
-require './person'
+require_relative 'person'
 
 # Student class demonstrating the inheritance principle
 class Student < Person
+  attr_accessor :classrooms
+
   def initialize(age, classrooms, name: 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classrooms = classrooms
```

Comment:
- [ ] I think that you should have a method that serves as the 'belongs-to' side (a student belongs to a classroom). Link to the article with examples: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships like :point_down: 
  
  ```ruby  
  def classroom=(classroom)
        @classroom = classroom
        classroom.students.push(self) unless classroom.students.include?(self)
    end
  ```

Reply:
Fixed.